EventFilter/Utilities/plugins/RawEventOutputModuleForBU.h

```diff
@@ -70,7 +70,8 @@ RawEventOutputModuleForBU<Consumer>::RawEventOutputModuleForBU(edm::ParameterSet
       instance_(ps.getUntrackedParameter<std::string>("ProductInstance", "")),
       token_(consumes<FEDRawDataCollection>(edm::InputTag(label_, instance_))),
       numEventsPerFile_(ps.getUntrackedParameter<unsigned int>("numEventsPerFile", 100)),
-      frdVersion_(ps.getUntrackedParameter<unsigned int>("frdVersion", 3)),
+      frdVersion_(ps.getUntrackedParameter<unsigned int>("frdVersion", 5)),
```

Comment:
why do you pick `5` and not `6` by default ?

Reply:
It could be 6 but then it doesn't work out of box with older CMSSW versions. If there is no concern about this, then let's make 6 default.

Reply:
If you mean that files produces with the new version of the output modules would not be readable by older CMSSW release, I don't think that's a concern.

The two use cases I know about are:
  - hilton tests, that re-build the files each time
  - hlt/gpu performance tests, for which we would like the new flag anyway

Reply:
ok, I'll push 6 as default in the commit I'm preparing.